src/resources/glossary.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Glossary
+description: A glossary reference for terminology used across dart.dev
+sitemap: false
+---
+
+The following are definitions of terminology used across the Dart documentation.
+
+## Irrefutable patterns
```

Comment:
I wonder if these entries should be singular? Seems most of the ones in the [Fuchsia Glossary](https://fuchsia.dev/fuchsia-src/glossary#terms) are.

```suggestion
## Irrefutable patterns
```